lwrunner/awsrunner.go

```diff
@@ -329,14 +324,10 @@ func (run AWSRunner) RunSSMCommandOnRemoteHost(cfg aws.Config, operation string)
 	)
 }
 
-// getAMIName takes an AMI image ID and an AWS region name as input
-// and calls the AWS API to get the name of the AMI. Returns the AMI
-// name or an error if unsuccessful.
-func getAMIName(amiImageId, region string) (string, error) {
-	cfg, err := config.LoadDefaultConfig(context.Background())
-	if err != nil {
-		return "", err
-	}
+// getAMIName takes an AMI image ID, an AWS region name, and an AWS
+// credential config as input and calls the AWS API to get the name
+// of the AMI. Returns the AMI name or an error if unsuccessful.
```

Comment:
This comment appears stale.

Reply:
It's updated in this commit, what did you notice that's stale about it?